terraform-common.mk

```diff
@@ -47,4 +47,4 @@ graph:
 	terraform graph -draw-cycles | dot -Tpng > graph.png
 
 $(ENV_NAME).tfvars:
-	$(TOP)/bin/generate-tfvars >$@
+	$(TOP)/bin/generate-tfvars $@
```

Comment:
Or did you mean it should be quoted to prevent issues with `$(ENV_NAME).tfvars` having spaces?